sleap/gui/app.py

```diff
@@ -65,6 +67,14 @@
 from sleap.gui.dialogs.formbuilder import FormBuilderModalDialog
 from sleap.gui.dialogs.metrics import MetricsTableDialog
 from sleap.gui.dialogs.shortcuts import ShortcutDialog
+
+# Open bulletin online if there is an ImportError (for MacOS)
+online_bulletin = False
+try:
+    from sleap.gui.dialogs.bulletin import BulletinWorker
+except ImportError:
+    online_bulletin = True
```

Comment:
```suggestion
ONLINE_BULLETIN = False
try:
    from sleap.gui.dialogs.bulletin import BulletinWorker
except ImportError:
    ONLINE_BULLETIN = True
```

Globals should be all caps